src/index.js

```diff
@@ -72,32 +72,32 @@ module.exports = class FederatedTypesPlugin {
   }
 
   importRemoteTypes() {
-    const remoteUrls = Object.values(this.remoteComponents).map((r) => {
-      let url;
+    const remoteUrls = Object.values(this.remoteComponents).map(r => {
```

Comment:
On a second thought, I think the `component name` collision you mentioned in the issue description still happen for the `non @` pattern.

So, perhaps we can pick the remote name from the `Remotes config` if the url doesn't follow `@` pattern.

Something like

```js
const remoteUrls = Object.entries(this.remoteComponents).map([remote, url] => {})
```

Reply:
Yeah it does. I’ll update this to reflect

Reply:
Alright that should do it now. Each declaration will get scoped to a specific folder based on the remote key